TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/middleware/clusters.py

```diff
@@ -58,7 +58,7 @@ def run_test_benchmarks(cluster: Cluster):
 
 # As a default we exclude system indices and searchguard indices
 def clear_indices(cluster: Cluster):
-    clear_indices_path = "/*,-.*,-searchguard*,-sg7*,.migrations_working_state"
+    clear_indices_path = "/*,-.*,-searchguard*,-sg7*,.migrations_working_state*"
```

Comment:
Nit: What happens if an RFS worker is running, will it die with an understandable explanation?

Reply:
By deleting the working state we could have running RFS workers do some very confusing things.  If we want to bucket this into 'advanced' options that we won't protect our users from that is ok with me, but I'd just like to be sure that is our expectation.